css-view-transitions-2/Overview.bs

```diff
@@ -264,6 +279,33 @@ Note: this event is fired when [=reveal document|revealing a document=].
 The <dfn attribute for=PageRevealEvent>viewTransition</dfn> [=getter steps=] are to return the
 [=inbound cross-document view-transition=] for [=this's=] [=relevant global object's=] [=associated document=].
 
+## Additions to {{Document}} ## {#additions-to-document-api}
+
+	<xmp class=idl>
+		dictionary StartViewTransitionOptions {
+			UpdateCallback? update = null;
+			sequence<DOMString>? types = null;
+		};
+
+		partial interface Document {
+			ViewTransition startViewTransition(optional StartViewTransitionOptions options);
+		};
+
+		callback UpdateCallback = Promise<any> ();
```

Comment:
The L1 spec already defines it: https://drafts.csswg.org/css-view-transitions-1/#callbackdef-updatecallback.

Reply:
Removed